### PR TITLE
Dependecy updates solr,psql,redis

### PR DESCRIPTION
**Pull request by Filip3mac**
Updating solr,redis,psql & other kubernetes modules.

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -267,20 +267,13 @@ redis:
   usePassword: false
 
 solr:  
-  # Please see all available overrides at https://github.com/helm/charts/tree/master/incubator/solr
-  # solr.enabled -- Flag to control whether to deploy SOLR
-  enabled: true
```
Comment (blazhovsky) [inline values.yaml:272]: we need to have the enabled flag for SOLR in the code

File: values.yaml [hunk 2]
```diff
@@ -289,10 +282,11 @@ solr:
     # solr.volumeClaimTemplates.storageSize -- Size of Solr PVC
     storageSize: 5Gi
   image:
+    registry: docker.io
     # solr.image.repository -- Repository for the SOLR image
-    repository: solr
+    repository: bitnami/solr
     # solr.image.tag -- Tag for the SOLR image
-    tag: 6.6.6
```
Comment (blazhovsky) [inline values.yaml:295]: this should still be 6.6.6 until we test how everything works with SOLR 8

File: templates/deployment.yaml [hunk 3]
```diff
@@ -107,6 +114,12 @@ spec:
                   key: ckanDatastoreReadUrl
             - name: CKAN_SOLR_URL
               value: {{ .Values.ckan.solr }}
+            - name: CKAN_SOLR_AUTH
```
Comment (Filip3mac) [inline templates/deployment.yaml:117]: Need to remove this

File: values.yaml [hunk 4]
```diff
@@ -17,6 +17,7 @@ pvc:
   enabled: true
   size: "1Gi"
   storageClassName: ""
+  accessmode: "ReadWriteMany"
```
Comment (Filip3mac) [inline values.yaml:20]: ReadWriteOnce default

File: values.yaml [hunk 5]
```diff
@@ -71,13 +72,15 @@ ckan:
   # ckan.siteId -- Site id
   siteId: "site-id-here"
   # ckan.siteUrl -- Url for the CKAN instance
-  siteUrl: "http://localhost:5000"
+  siteUrl: "http://localhost:5003"
```
Comment (Filip3mac) [inline values.yaml:75]: 5000

**Review by KirilPoposki98 — COMMENTED** [review]
Ready for merge

**Review by KirilPoposki98 — APPROVED** [review]

**Pull request merged**